content/content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesAction.java

```diff
@@ -4215,7 +4215,7 @@ public String buildListContext (	VelocityPortlet portlet,
 			// notshow the public option or notification when in dropbox mode
 			context.put("dropboxMode", Boolean.TRUE);
 			// allow filtering of dropboxes by group (SAK-14625)
-			String collectionId = (String) state.getAttribute (STATE_COLLECTION_ID);
```

Comment:
Hi, Earl, excuse the delay, I have been out of work.
I don't know what do you mean with "update". When I do a git push from my local branch, it says "Everything up-to-date". Am I missing something?
Sorry, I will try to be more clean with my comments.